src/org/labkey/test/components/glassLibrary/components/ReactSelect.java

```diff
@@ -303,6 +319,7 @@ public ReactSelect select(String option)
                 .open()
                 .clickOption(option)
                 .getSelections();
+        waitForClosed();
 
         // TODO Comment out this line for the time being. Issue 37897: Sample Management: Lookup field values are not consistently displayed between the edit sample panel and the create sample panel.
 //        assertTrue("Expected '" + option + "' to be selected.  Current selections: " + selections, selections.contains(option));
```

Comment:
Not part of your change, but I think these two commented lines should go away. 
Ideally this should be a control used in other apps not just SM, so the first comment is not appropriate. And the bug referenced is TBD so who knows when it would get fixed.
Second I like that there is an assert in the control that checks to make sure it did wha tit was asked to do, but if it can't do that reliably for everyone it shouldn't do it.

Reply:
Removed. Additionally, such validation isn't valid for all selects. In the the SM permissions page, the selected items show up elsewhere on the page, not in the select itself.